### ISSUE TITLE
Update Ignite support version

### ISSUE THREAD

**Issue by chenjian2664** [trinodb/trino#16380]
In Ignite official docs https://ignite.apache.org/docs/latest already removed Ignite 2.8.0 support.

**Comment by chenjian2664** [comment]
While doing local test, Ignite 2.8 encountered OOM sometimes:
```
Exception in thread "client-connector-#61" java.lang.OutOfMemoryError: GC overhead limit exceeded
        at java.util.regex.Pattern.compile(Pattern.java:1689)
        at java.util.regex.Pattern.<init>(Pattern.java:1351)
        at java.util.regex.Pattern.compile(Pattern.java:1028)
        at java.lang.String.replaceAll(String.java:2223)
        at org.apache.ignite.internal.processors.odbc.SqlListenerUtils.translateSqlWildcardsToRegex(SqlListenerUtils.java:268)
        at org.apache.ignite.internal.processors.query.QueryUtils.matches(QueryUtils.java:1545)
        at org.apache.ignite.internal.processors.query.h2.IgniteH2Indexing.lambda$tablesInformation$3(IgniteH2Indexing.java:1801)
        at org.apache.ignite.internal.processors.query.h2.IgniteH2Indexing$$Lambda$279/1048495342.test(Unknown Source)
        at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:174)
        at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:175)
        at java.util.concurrent.ConcurrentHashMap$ValueSpliterator.forEachRemaining(ConcurrentHashMap.java:3566)
        at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:482)
        at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:472)
        at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:151)
        at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:174)
        at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
        at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:418)
        at org.apache.ignite.internal.processors.query.h2.IgniteH2Indexing.tablesInformation(IgniteH2Indexing.java:1822)
        at org.apache.ignite.internal.processors.odbc.jdbc.JdbcMetadataInfo.getTablesMeta(JdbcMetadataInfo.java:155)
        at org.apache.ignite.internal.processors.odbc.jdbc.JdbcRequestHandler.getTablesMeta(JdbcRequestHandler.java:1076)
        at org.apache.ignite.internal.processors.odbc.jdbc.JdbcRequestHandler.doHandle(JdbcRequestHandler.java:334)
        at org.apache.ignite.internal.processors.odbc.jdbc.JdbcRequestHandler.handle(JdbcRequestHandler.java:247)
        at org.apache.ignite.internal.processors.odbc.ClientListenerNioListener.onMessage(ClientListenerNioListener.java:195)
        at org.apache.ignite.internal.processors.odbc.ClientListenerNioListener.onMessage(ClientListenerNioListener.java:49)
        at org.apache.ignite.internal.util.nio.GridNioFilterChain$TailFilter.onMessageReceived(GridNioFilterChain.java:279)
        at org.apache.ignite.internal.util.nio.GridNioFilterAdapter.proceedMessageReceived(GridNioFilterAdapter.java:109)
        at org.apache.ignite.internal.util.nio.GridNioAsyncNotifyFilter$3.body(GridNioAsyncNotifyFilter.java:97)
        at org.apache.ignite.internal.util.worker.GridWorker.run(GridWorker.java:120)
        at org.apache.ignite.internal.util.worker.GridWorkerPool$1.run(GridWorkerPool.java:70)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
```

**Comment by chenjian2664** [comment]
> While doing local test, Ignite 2.8 encountered OOM sometimes:
> 
> ```
> Exception in thread "client-connector-#61" java.lang.OutOfMemoryError: GC overhead limit exceeded
>         at java.util.regex.Pattern.compile(Pattern.java:1689)
>         at java.util.regex.Pattern.<init>(Pattern.java:1351)
>         at java.util.regex.Pattern.compile(Pattern.java:1028)
>         at java.lang.String.replaceAll(String.java:2223)
>         at org.apache.ignite.internal.processors.odbc.SqlListenerUtils.translateSqlWildcardsToRegex(SqlListenerUtils.java:268)
>         at org.apache.ignite.internal.processors.query.QueryUtils.matches(QueryUtils.java:1545)
>         at org.apache.ignite.internal.processors.query.h2.IgniteH2Indexing.lambda$tablesInformation$3(IgniteH2Indexing.java:1801)
>         at org.apache.ignite.internal.processors.query.h2.IgniteH2Indexing$$Lambda$279/1048495342.test(Unknown Source)
>         at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:174)
>         at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:175)
>         at java.util.concurrent.ConcurrentHashMap$ValueSpliterator.forEachRemaining(ConcurrentHashMap.java:3566)
>         at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:482)
>         at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:472)
>         at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:151)
>         at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:174)
>         at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
>         at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:418)
>         at org.apache.ignite.internal.processors.query.h2.IgniteH2Indexing.tablesInformation(IgniteH2Indexing.java:1822)
>         at org.apache.ignite.internal.processors.odbc.jdbc.JdbcMetadataInfo.getTablesMeta(JdbcMetadataInfo.java:155)
>         at org.apache.ignite.internal.processors.odbc.jdbc.JdbcRequestHandler.getTablesMeta(JdbcRequestHandler.java:1076)
>         at org.apache.ignite.internal.processors.odbc.jdbc.JdbcRequestHandler.doHandle(JdbcRequestHandler.java:334)
>         at org.apache.ignite.internal.processors.odbc.jdbc.JdbcRequestHandler.handle(JdbcRequestHandler.java:247)
>         at org.apache.ignite.internal.processors.odbc.ClientListenerNioListener.onMessage(ClientListenerNioListener.java:195)
>         at org.apache.ignite.internal.processors.odbc.ClientListenerNioListener.onMessage(ClientListenerNioListener.java:49)
>         at org.apache.ignite.internal.util.nio.GridNioFilterChain$TailFilter.onMessageReceived(GridNioFilterChain.java:279)
>         at org.apache.ignite.internal.util.nio.GridNioFilterAdapter.proceedMessageReceived(GridNioFilterAdapter.java:109)
>         at org.apache.ignite.internal.util.nio.GridNioAsyncNotifyFilter$3.body(GridNioAsyncNotifyFilter.java:97)
>         at org.apache.ignite.internal.util.worker.GridWorker.run(GridWorker.java:120)
>         at org.apache.ignite.internal.util.worker.GridWorkerPool$1.run(GridWorkerPool.java:70)
>         at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
>         at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
>         at java.lang.Thread.run(Thread.java:748)
> ```

Related to https://issues.apache.org/jira/browse/IGNITE-12687, fixed in Ignite 2.9.0 and above

**Comment by chenjian2664** [comment]
Ignite exposed `statistics` related infos under `sys` schema from 2.12.0:
`STATISTICS_CONFIGURATION` , `STATISTICS_LOCAL_DATA`, `STATISTICS_PARTITION_DATA`. 
There also has a table `STATISTICS_GLOBAL_DATA` even do not docs yet.
